sdks/wasm/DebuggerTestSuite/Tests.cs

```diff
@@ -77,7 +77,7 @@ void CheckLocation (string script_loc, int line, int column, Dictionary<string,
 
 				var bp1_res = await SetBreakpoint ("dotnet://debugger-test.dll/debugger-test.cs", 5, 2, ctx);
 
-				Assert.Equal ("dotnet:0", bp1_res.Value ["breakpointId"]);
+				Assert.EndsWith (bp1_res.Value ["breakpointId"]?.ToString (), bp1_res.Value ["breakpointId"].ToString());
```

Comment:
What is this doing?

Reply:
The breakpointId is now created by the target and no longer follows has the dotnet: scheme

Reply:
Ahhh, I didn't actually read the code